packages/server/src/artifact/checkin/output.rs

```diff
@@ -781,6 +784,32 @@ impl Server {
 		output: Arc<RwLock<Graph>>,
 		visited: &mut BTreeSet<tg::Path>,
 		progress: &ProgressState,
+	) -> tg::Result<()> {
+		self.copy_or_move_all_inner(&dest,  output, visited, progress)
+			.await?;
+		self.set_file_times_to_epoch(dest).await?;
+		Ok(())
+	}
+
+	async fn copy_or_move_all_inner(
+		&self,
+		dest: &tg::Path,
+		output: Arc<RwLock<Graph>>,
+		visited: &mut BTreeSet<tg::Path>,
+		progress: &ProgressState,
+	) -> tg::Result<()> {
+		self.copy_or_move_all_inner(&dest, output, visited, progress)
+			.await?;
+		self.set_file_times_to_epoch(dest).await?;
```

Comment:
Do we need this now that the call at the end is guaranteed to recurse?

Reply:
You're right- removed and confirmed it still works as intended